bot.js

```diff
@@ -1,41 +1,42 @@
 const Discord = require('discord.js');
-const client = new Discord.Client();
 const fs = require('fs');
-const prefix = 'n.';
-const config = require('./config.json');
-const { token } = config;
+const { token } = require('./config.json');
 
+const client = new Discord.Client();
 client.commands = new Discord.Collection();
+
+const prefix = 'n.';
+
+
+
+// load commands
 const commandFiles = fs.readdirSync('./commands/').filter(file => file.endsWith('.js'));
 for (const file of commandFiles) {
 	const command = require(`./commands/${file}`);
 
 	client.commands.set(command.name, command);
 }
 
+
+
+
+/// process
 client.once('ready', () => {
 	console.log('The Culture Man is alive');
 });
 
+
 client.on('message', message => {
-	if (!message.content.startsWith(prefix) || message.author.bot) return;
+	if (!message.content.startsWith(prefix) || message.author.bot) {
+		return;
+	}
 
 	const args = message.content.slice(prefix.length).split(/ +/);
 	const command = args.shift().toLowerCase();
 
-	if (command === 'culture') {
-		client.commands.get('culture').execute(message, args);
-	} else if (command == 'search') {
-		client.commands.get('search').execute(message, args);
-	} else if (command == 'help') {
-		client.commands.get('help').execute(message, args);
-	} else if (command == 'read') {
-		client.commands.get('read').execute(message, args);
-	} else if (command == 'random') {
-		client.commands.get('random').execute(message, args);
-	} else if (command == 'tags') {
-		client.commands.get('tags').execute(message, args);
+	if (client.commands.has(command)) {

```

Comment:
How would more than one command be added? ( I do not have any coding experience, I used a lot of Stackoverflow for this bot )

Reply:
you first read the files and
collect on `client.commands`

at last just check the command exist on `client.commands`
if was execute
`if (client.commands.has(command)) {`

and for adding command need make a new commands file on commands folder

or you want do more than one command at time

Reply:
and for warning I do not have complete experience with develop on "discord".